app/configuration.py

```diff
@@ -34,12 +35,14 @@
 
 @dataclass
 class Coil:
+    input_type: ClassVar[str] = "COIL"
```

Comment:
for the non-python-expert out there. What does this `ClassVar` do?

Reply:
It simply sets the specified value for the `input_type` field for all objects in the class. I've never needed such a thing before and had to go look up how to do it, but I think it's a nice simple solution here. We could just check the type of the config object, but  I prefer `if message.input_type == 'COIL'` to `if isinstance(message.configuration, Coil)`

Reply:
OK, that's interesting. Can you explain why do you prefer to do string comparison rather than type checks?

Reply:
Assuming the string comes from a constant value and is checked at compile time, I just think it's much easier to read as a string comparison